10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
 
 \section{Sys/sys}
 \label{std-sys}
+
+\section{Remoting}
+\label{std-remoting}
+
+Haxe remoting is a way to talk between different platforms. With Haxe remoting we can communicate transparently, send data and call methods, between server and client side.
```

Comment:
It can't be a way to "talk". Maybe "Is a way to transmit data..." or something? Same goes for "we can communicate". Its not the user communicating, its the applications, no?